website/content/docs/api-clients/api.mdx

```diff
@@ -79,3 +79,169 @@ The following method conventions are used within Boundary's API:
+Some example configurations are listed below.
+
+### Rate limiting configuration examples
+
+The following example shows a simple configuration where the same limits are applied to all resources and examples:
+
+```hcl
+controller {
+  // total limit for all resources and actions
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["*"]
+	per  	= "total"
+	limit 	= 500
+	period	= "1s"
+  }
+
+  // limit for ip addresses to all resources+actions to prevent a malicious
+  // host that is fabricating tokens, or spamming unauthed endpoints
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["*"]
+	per 	= "ip-address"
+	limit 	= 100
+	period	= "1s"
+  }
+
+  // Limit of all authed requests, to prevent one user consuming all of the total limit
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["*"]
+	per 	= "auth-token"
+	limit 	= 100
+	period	= "1s"
+  }
+
+  // other controller configuration
+}
+```
+
+The following example shows a configuration that disables rate limiting.
+You may want to disable rate limiting if you already use an external system like a reverse proxy to apply rate limiting:
+
+```hcl
+controller {
+  api_rate_limit_disable = true
+
+  // other controller configuration
+}
+```
+
+The following example uses the default settings for most endpoints, but configures a single override:
+
+```hcl
+controller {
+  api_rate_limit {
+	resources = ["target"]
+	actions   = ["list"]
+	per 	= "auth-token"
+	limit 	= 10
+	period	= "1s"
+  }
+
+  // other controller configuration
+}
+```
+
+The following example is more complex.
+Initially it sets some defaults to apply to all resources and actions.
+Then it configures some specific endpoints with different limits.
+
+```hcl
+controller {
+  // total limit for all resources and actions
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["*"]
+	per 	= "total"
+	limit 	= 500
+	period	= "1s"
+  }
+
+  // total limit for all list operations
+  // sets an overall cap on list since it is relatively expensive.
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["list"]
+	per 	= "total"
+	limit 	= 200
+	period	= "1s"
+  }
+
+  // limit for ip addresses to all resources+actions to prevent a malicious
+  // host that is fabricating tokens, or spamming unauthed endpoints
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["*"]
+	per 	= "ip-address"
+	limit 	= 50
+	period	= "1s"
+  }
+
+  // Limit of all authed requests, this essentially sets a default for
+  // all authed requests to prevent one user consuming all of the total limit
+  // set above. Any limits that follow can override the default for
+  // specific sets of resources/actions
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["*"]
+	per 	= "auth-token"
+	limit 	= 100
+	period	= "1s"
+  }
+
+  // Limit of all list operations by auth token to set a sane default since
+  // they are generally more expensive.
+  api_rate_limit {
+	resources = ["*"]
+	actions   = ["list"]
+	per 	= "total"
```

Comment:
I think this should be `auth_token`, since that's what the description of this section states, and there's already a limit on `total` above

```suggestion
	per 	= "auth-token"
```

Reply:
> I think this should be `auth_token`, since that's what the description of this section states, and there's already a limit on `total` above

Yep, I think you are correct based on what the comment above it says. Nice catch!